### PR TITLE
Add OSGi support (#46)

### PR DESCRIPTION
**Pull request by robertcsakany**

### CODE REVIEW

**Comment by blackbelt-oss** [comment]
There is some problem in the travis build. The last version on master was built is not correct, becuase the JUnit test there was not runned.

File: src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java [hunk 1]
```diff
@@ -11,6 +11,11 @@
 
 class ThrowingBiConsumerTest {
 
+
```
Comment (pivovarit) [inline src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: To be honest, I have no clue what is it supposed to test. It's definitely not testing anything from the library... will remove it now.

Comment (pivovarit) [reply src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java]: https://github.com/pivovarit/throwing-function/commit/650f721de7149bf04d6115a1bf2ecbe778c0e125

**Comment by pivovarit** [comment]
Thank you :)

**Pull request merged**